tests/lib/nested.sh

```diff
@@ -167,8 +167,8 @@ nested_uc20_transition_to_system_mode() {
 
 nested_prepare_ssh() {
     if nested_is_core_ge 24; then
-        remote.exec "sudo useradd --uid 12345 --extrausers test"
-        remote.exec "sudo useradd --extrausers external"
+        remote.exec "sudo useradd --uid 12345 -m --extrausers test"
```

Comment:
Nitpick, I'd expand -m to --create-home so that it's more obvious what's going on.